src/controllers/dashboard/quickconnect.js

```diff
@@ -0,0 +1,58 @@
+import loading from 'loading';
+import toast from 'toast';
+import globalize from 'globalize';
+
+/* eslint-disable indent */
```

Comment:
New files should have proper indentation. You probably saw this in something that had been migrated to avoid really large diffs.